### PR TITLE
Production readiness: showcase redesign, publish fixes, OSS standards, Peak sample

### PR DESCRIPTION
**Pull request by thisAAY**
## Summary

Gets Arkive ready for a public 0.0.2 release: a redesigned web showcase, a repaired Maven Central publish setup, Infinum open-source standards with CI and a GitHub Pages demo, and a component-rich sample app.

**Related issue**: None

### Type

- [x] **Feature**: This pull request introduces a new feature.
- [x] **Bug fix**: This pull request fixes a bug.
- [x] **Refactor**: This pull request refactors existing code.
- [x] **Documentation**: This pull request updates documentation.

#### Additional information

- [x] This pull request introduces a **breaking change**.

### Description

**Web showcase redesign** — the 3-page template is replaced by a single-page vanilla-JS catalogue (Infinum design, system fonts, no dependencies): sidebar module/group tree, grid/list overview, ⌘K fuzzy search palette, and a detail view with pan/zoom canvas, device-frame mode, variant compare, and Figma links. Hash routing keeps it GitHub-Pages friendly; the Arkive mark is the favicon.

**Publish setup fixes** — the March deploy published the plugin marker at version `unspecified` pointing at `Arkive:plugin:unspecified`, so the plugin id could never resolve from Maven Central. Fixed alongside: real POM metadata replacing template placeholders, a Dokka source-set workaround unblocking javadoc jars (Kotlin/dokka#3701), the plugin version stamped into the jar instead of hardcoded in three places, `project(":metadata")` instead of the published artifact, and signing required only for Sonatype deploys. Deploying 0.0.2 fixes Central resolution going forward.

**OSS standards + CI** — `.github` aligned with kmp-prince-of-versions: quality-check workflows (detekt, library builds and tests, sample build as a real plugin consumer), dependency submission, Renovate, CODEOWNERS, form-based issue templates. A fresh checkout is CI-buildable via the `-PskipSample` bootstrap. On merge to main, the sample showcase deploys to GitHub Pages (requires one-time repo setting: Pages → Source: GitHub Actions).

**Plugin behavior** — snapshots are now recorded with a transparent window background instead of the dark default theme; a `keepSnapshots` flag (default true) copies instead of moves goldens so `verifyPaparazzi` keeps working; `disablePreviewParameters` is renamed to `enablePreviewParameters` (default true) for consistent enable-semantics — **breaking** for the extension DSL.

**Peak sample** — the sample is rebuilt as an outdoor trail tracker with 29 annotated composables across 8 component groups plus 4 full screens, generating a catalogue of 31 components and 324 variant snapshots (goldens committed for the verify workflow).

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have tested my changes, including edge cases.
- [ ] I have added necessary tests for the changes introduced (if applicable).
- [x] I have updated the documentation to reflect my changes (if applicable).

## Additional notes

After merge: enable GitHub Pages (Source: GitHub Actions) and run `./gradlew deployAll` with Sonatype credentials to publish 0.0.2 to Maven Central. The `unspecified` marker on Central becomes inert once 0.0.2 exists; Sonatype support can remove it cosmetically.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://github.com/user-attachments/assets/2f17c379-d966-455f-9757-fc60da0b1325

### CODE REVIEW

File: processor/src/main/java/com/infinum/arkive/processor/specs/ViewSpec.kt [hunk 1]
```diff
@@ -44,12 +44,12 @@ class ViewSpec(
...
-             ${getCodeBody(holder).toString().trimIndent()}
-            } 
-        """.trimIndent()
+    private fun getRunnerFunction(holder: ViewHolder): CodeBlock {
+        return CodeBlock.builder().apply {
+            beginControlFlow("runner(%S)", holder.functionId)
+            add(getCodeBody(holder))
+            endControlFlow()
+        }.build()
     }
```
Comment (spasepremcheski) [inline processor/src/main/java/com/infinum/arkive/processor/specs/ViewSpec.kt:53]: Can the try-catch be replicated as in `ComposeRunnerSpec` 
```
private fun getRunnerFunction(holder: ComposeHolder, wrapperSuffix: String): CodeBlock {
        val functionRunner = MemberName(
            packageName = "com.infinum.arkive",
            simpleName = "${holder.functionId}$wrapperSuffix",
        )
        // Each component (base or its variants) is isolated: a preview that fails to render
        // logs and is dropped from the showcase instead of aborting the whole snapshot run.
        return CodeBlock.builder().apply {
            beginControlFlow("try")
            addStatement("%M(runner)", functionRunner)
            nextControlFlow("catch (e: %T)", ClassName("kotlin", "Throwable"))
            addStatement(
                "println(%S + e.message)",
                "Arkive: skipping component ${holder.functionId}$wrapperSuffix, snapshot failed: ",
            )
            endControlFlow()
        }.build()
    }
```

**Review by spasepremcheski — APPROVED** [review]
